src/alohalytics.h

```diff
@@ -95,6 +98,9 @@ class Stats final {
 
   // Uploads all previously collected data to the server.
   void Upload(TFileProcessingFinishedCallback upload_finished_callback = TFileProcessingFinishedCallback());
+
+  // Collects data blobs for uploading to the server.
```

Comment:
// Calls |result_callback| on the blobs that are going to be uploaded to the server.